packages/gasket-plugin-express/docs/setup.md

```diff
@@ -106,10 +106,29 @@ module.exports = {
   }
 }
 ```
+You can configure which paths middleware will run on by adding the middleware configuration array to your local `gasket.config.js`. The array is made up of objects with the name of the middlware (Gasket plugin name) you want to configure and an array of path patterns representing the paths to match for this middleware. Pattern matching entries in the array can come in the form of path strings, path pattern strings, and/or regular expressions. Pattern matching logic follows Express's [app.use pattern matching].
+
+```js
+  middleware: [
+    {
+      plugin:'api-middleware', // Name of the Gasket plugin
```

Comment:
To be communicate clearly that these are Gasket plugins names, could you follow the [plugin naming convention](https://github.com/godaddy/gasket/blob/main/packages/gasket-resolve/README.md#naming-convention) for your examples, such as:
- gasket-plugin-example
- @some/gasket-plugin-example
- @another/gasket-plugin-example